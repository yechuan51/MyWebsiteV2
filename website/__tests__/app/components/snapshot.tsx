import NavBarComp, { NavBarItemType } from "@/components/main_nav_bar/navBar";
import { render } from "@testing-library/react";

it("render unchanged", () => {
  // First, create a list of two dummy items for NavBarCompPropsType.
  const items: NavBarItemType[] = [
    { key: "item1", displayText: "Item 1", href: "/item1" },
    { key: "item2", displayText: "Item 2", href: "/item2" },
  ];
  // Create a NavBarComp with the items using render function.
  const { asFragment } = render(<NavBarComp items={items} />);
  // expect the result to match snapshot.
  expect(asFragment()).toMatchSnapshot();
});
