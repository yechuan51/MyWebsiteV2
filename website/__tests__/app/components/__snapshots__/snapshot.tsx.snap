// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render unchanged 1`] = `
<DocumentFragment>
  <nav
    class="flex flex-row justify-between items-center py-4 px-6 bg-white border-b border-gray-400"
  >
    <p>
      <a
        href="/"
        title="Logo"
      >
        My website
      </a>
    </p>
    <ul
      class="flex flex-row items-center"
    >
      <li
        class="mr-4 font-sfPro"
      >
        <a
          class="hover:bg-gray-200 text-xl px-2.5 py-5"
          href="/item1"
        >
          Item 1
        </a>
      </li>
      <li
        class="mr-4 font-sfPro"
      >
        <a
          class="hover:bg-gray-200 text-xl px-2.5 py-5"
          href="/item2"
        >
          Item 2
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
